tests/test_standalone_stac_item.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+import requests
+import xncml
+
+from STACpopulator.implementations.CMIP6_UofT.add_CMIP6 import (
+    CMIP6ItemProperties,
+    make_cmip6_item_id,
+)
+from STACpopulator.models import GeoJSONPolygon
+from STACpopulator.stac_utils import STAC_item_from_metadata
+
+
+def test_standalone_stac_item():
+    url = (
+        "https://pavics.ouranos.ca/twitcher/ows/proxy/"
+        "thredds/ncml/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc"
+        "?catalog=https%3A%2F%2Fpavics.ouranos.ca%2Ftwitcher%2Fows%2Fproxy%2F"
+        "thredds%2Fcatalog%2Fbirdhouse%2Ftestdata%2Fxclim%2Fcmip6%2Fcatalog.html"
+        "&dataset=birdhouse%2Ftestdata%2Fxclim%2Fcmip6%2Fsic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc"
+    )
```

Comment:
Should use `@pytest.mark.parametrize` and test for multiple variations of this URL (combinations with `thredds/ncml/`, `thredds/catalog/`, `.xml`, `.html`, with/without the `catalog` query parameter). 

In each case, it should always result in the same STAC Item.